test/test_dynamic_shapes.py

```diff
@@ -1519,6 +1434,7 @@ def test_dim_constraints_reduce_inequalities_error(self):
         ):
             dcp.doprint(answer)
 
+    @unittest.expectedFailure  # avik, you are our only hope!
```

Comment:
This is in the offline solver cluster of issues

Reply:
This is the same issue as the other I got tagged in, export's solver/sympy can't handle CleanDiv.

Reply:
It looks like the first two resolved this one